modules/ixBidAdapter.js

```diff
@@ -68,12 +73,14 @@ function bidToVideoImp(bid) {
     }
   }
 
-  for (let adUnitProperty in videoAdUnitRef) {
-    if (videoAdUnitWhitelist.indexOf(adUnitProperty) !== -1 && !imp.video.hasOwnProperty(adUnitProperty)) {
+  for (const adUnitProperty in videoAdUnitRef) {
+    if (videoAdUnitAllowlist.indexOf(adUnitProperty) !== -1 && !imp.video[adUnitProperty]) {
```

Comment:
could there be cases where the `imp.video[adUnitProperty]` value is falsey?

Like a couple in your videoAdUnitAllowlist for example look like they might be at times:

`skip` (boolean?) if this is false?
`boxingallowed` ? is this a boolean? if false do you want to pass along?
`startdelay` could this be set to 0 (zero) ?

hasOwnProperty is better in this case, otherwise if you do not care about passing along falsey values, then this change can stay.
I do not actually know if these are legit values on the imp.video object, but want to ask and make sure you are aware.

These two above examples would be

Reply:
Yes, good catch.

Reply:
added.